content.js

```diff
@@ -1,17 +1,23 @@
 const myAudio = new Audio(chrome.runtime.getURL("ding.mp3"));
 
+
+
 const init = () => {
   const entireBody = document.body.innerText;
   // So it can match one of those messages: 
     // - "No delivery windows available", 
     // - "No attended delivery windows are available"
     // - "No doorstep delivery windows are available";   
-  if (entireBody.search(/No(.*)delivery windows(.*)are available/) !== -1) {
```

Comment:
![Screen Shot 2020-04-11 at 11 34 14 PM](https://user-images.githubusercontent.com/493808/79059857-fbad0380-7c4c-11ea-814f-ebb9fc786d3e.png)

Reply:
I think there's some crazy A/B testing going on. I still get the original text. This is pretty good.

I was thinking what I actually wanted to do was search for buttons that match "actual time slots", but I didn't have a proper body saved of actual availability when I first wrote the script...